src/main/resources/application-prod.yml

```
@@ -18,4 +21,13 @@ spring:
-    database-platform: org.hibernate.dialect.PostgreSQLDialect
\ No newline at end of file
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
+
+imagekit:
+  publicKey: ${imageKit_PUBLIC_KEY}
+  privateKey: ${imageKit_PRIVATE_KEY}
+  urlEndpoint: ${imageKit_URL_ENDPOINT}
+  tokenImageKitAuthUUID: ${tokenImageKitAuthUUID}
+
+cors:
+  allowed-origins: ${CORS_ALLOWED_ORIGINS}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)